### ISSUE TITLE
PyCUDA

### ISSUE THREAD

**Issue by jakirkham** [ContinuumIO/anaconda-recipes#111]
Would be nice to have PyCUDA added to `defaults`. Given it links against CUDA, this may be the only way to distribute it ATM. Though if there is a way to get it into conda-forge as well, would be happy to discuss over at issue ( https://github.com/conda-forge/staged-recipes/issues/3178 ).

**Comment by jakirkham** [comment]
Copying you on this, @jjhelmus, in case you have any ideas on how to approach this.

**Comment by jjhelmus** [comment]
I don't believe there are any plans for adding PyCUDA to` defaults` at this time.  Do you have a working recipe for the package?  That would make the process of including it in `defaults` considerably easier.

**Comment by jakirkham** [comment]
I see. Thanks for the info.

We have a recipe in PR ( https://github.com/conda-forge/staged-recipes/pull/1775 ). However I don't know whether or not it works as we don't have `nvcc`. So we can't build it at `conda-forge` currently. Maybe @inducer can advise? 😉

**Comment by inducer** [comment]
You hit the nail on the head. At first I was excited when I saw that `cudatoolkit` was packaged, but when I realized that that did not include `nvcc`, I gave up for the time being.

**Comment by jakirkham** [comment]
Reading between the lines a bit, I think [these docker images]( https://github.com/jjhelmus/docker-images ) are used to build things with CUDA support. They are basically like `condaforge/linux-anvil` used in `conda-forge`, but with a different base image (e.g. `nvidia/cuda:7.5-cudnn5-devel-centos6`).

We tried to do the same thing some time ago in conda-forge. For instance PR ( https://github.com/conda-forge/docker-images/pull/21 ) is very similar (though outdated a bit), but there were concerns about whether packages could leave the Docker images and what was allowed to leave. Though I suppose we could just start that effort again given that Continuum has likely worked out the tricky points regarding redistribution with NVIDIA directly. Just unclear on what terms conda-forge could redistribute such things.
